templates/test/overview.html.ep

```diff
@@ -1,6 +1,7 @@
 % layout 'bootstrap';
 % title 'Test summary';
 % use OpenQA::Schema::Result::Jobs;
+% use OpenQA::Utils;
```

Comment:
if you need openQA::Utils in a template it's a clear indicator that you put a render helper in the wrong file